tests/test_fields.py

```diff
@@ -159,50 +161,52 @@ def test_filter_indexed_string(self):
         with session.Session() as s:
             dst = s.open_dataset(bio, "w", "src")
             hf = dst.create_dataframe('src')
-            strings = ['a', 'bb', 'ccc', 'dddd']
-            f = fields.IndexedStringImporter(s, hf, 'foo')
-            f.write(strings)
-            f = s.get(hf['foo'])
-            self.assertListEqual([0, 1, 3, 6, 10], f.indices[:].tolist())
+            data = ['a', 'bb', 'ccc', 'dddd']
+            indices, values, offsets, written_row_count = utils.one_dim_data_to_indexed_for_test(data, 10)               
+            foo = fi.IndexedStringImporter(s, hf, 'foo')
+            foo.transform_and_write_part(indices, values, offsets, 0, written_row_count)
+
+            self.assertListEqual([0, 1, 3, 6, 10], hf['foo'].indices[:].tolist())
 
             f2 = s.create_indexed_string(hf, 'bar')
-            s.apply_filter(np.asarray([False, True, True, False]), f, f2)
+            s.apply_filter(np.asarray([False, True, True, False]), hf['foo'], f2)
             self.assertListEqual([0, 2, 5], f2.indices[:].tolist())
             self.assertListEqual([98, 98, 99, 99, 99], f2.values[:].tolist())
             self.assertListEqual(['bb', 'ccc'], f2.data[:])
             self.assertEqual('bb', f2.data[0])
             self.assertEqual('ccc', f2.data[1])
 
+
     def test_reindex_indexed_string(self):
         bio = BytesIO()
         with session.Session() as s:
             dst = s.open_dataset(bio, "w", "src")
             hf = dst.create_dataframe('src')
-            strings = ['a', 'bb', 'ccc', 'dddd']
-            f = fields.IndexedStringImporter(s, hf, 'foo')
-            f.write(strings)
-            f = s.get(hf['foo'])
-            self.assertListEqual([0, 1, 3, 6, 10], f.indices[:].tolist())
+            data = ['a', 'bb', 'ccc', 'dddd']
```

Comment:
This test wasn't really intended for testing importing; I'm not sure why I used an importer at the time. Do we have a set of tests that explicitly tests the importers; one per type?

Reply:
This fix is mainly to fix the failed unittest, not for explicitly tests the importers.

The unittests in test_importer.py and test_parsers.py is explicitly testing the importers. Think that looks sufficient to me. Would you like to check if it's sufficient?